src/components/can_cooperation/src/can_module.cc

```diff
@@ -242,9 +242,11 @@ functional_modules::ProcessResult CANModule::HandleMessage(
                  == function_name) {
         if (value.isMember(json_keys::kParams)) {
           if (value[json_keys::kParams].isMember(message_params::kDevice)) {
+            // TODO(KKolodiy): take out "input" from message
```

Comment:
Is this implemented from HMI side? Should be implemented both on HMI and here before accepting pull request